options/ansi/generic/stdio-stubs.cpp

```diff
@@ -495,12 +495,12 @@ static int do_scanf(H &handler, const char *fmt, __gnuc_va_list args) {
                             if (c >= '0' && c <= '9') {
                                 handler.consume();
                                 res = res * 16 + (c - '0');
-                            } else if (c >= 'a' && c <= 'f') {
+                            } else if (c >= 'a' && c <= 'f' + 10) {
                                 handler.consume();
                                 res = res * 16 + (c - 'a');
-                            } else if (c >= 'A' && c <= 'F') {
+                            } else if (c >= 'A' && c <= 'F' + 10) {
```

Comment:
These +10 are in the wrong spot?

Reply:
Oh yeah, they should be in the actual addition right?

Reply:
Yeah